tools/traffic/metrics/metrics.go

```diff
@@ -0,0 +1,91 @@
+package metrics
+
+import (
+	"fmt"
+	"github.com/Layr-Labs/eigensdk-go/logging"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/collectors"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"net/http"
+)
+
+// Metrics allows the creation of metrics for the traffic generator.
+type Metrics interface {
```

Comment:
I see. So traffic generator will have one set of each type of metrics, and each metric is identified by the label

Reply:
Yes, that's the way the code currently works. If you'd prefer that I instead use unique names instead of labels, let me know (not a tricky change).